public/js/strings.json

```diff
@@ -10,6 +11,8 @@
   "editor.removeBreakpoint": "Remove Breakpoint",
   "editor.editBreakpoint": "Edit Breakpoint",
   "editor.addConditionalBreakpoint": "Add Conditional Breakpoint",
+  "editor.conditionalPanel.placeholder": "This breakpoint will pause when the expression is true",
+  "expressions.placeholder": "Add Watch Expression",
```

Comment:
looks like the tests are failing because `strings.json` needs `loadingText` from the properties.

It's interesting that it's using it... but i can investigate that later :)